test/srt/test_pixtral.py

```diff
@@ -0,0 +1,183 @@
+"""
+python3 -m unittest test_pixtral
```

Comment:
add this to per-commit CI https://github.com/sgl-project/sglang/blob/dc3bee4815183d19761082b9ac90ada72d0151ef/test/srt/run_suite.py#L7

Reply:
added